src/main/java/com/google/api/codegen/viewmodel/testing/MockCombinedView.java

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel.testing;
+
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.api.codegen.viewmodel.FileHeaderView;
+import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.auto.value.AutoValue;
+import java.util.List;
+
+/**
+ * Sort of a union for mock-related classes. Used in languages that idiomatically put mocks and
+ * tests in the same file.
```

Comment:
Reworded. The tests are "written" as a TODO for the time being. I will land them in a separate PR to keep the PR small.